keyboards/doio/kb19/keymaps/default/keymap.c

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2022 DOIO
+ * Copyright 2022 DOIO2022 <https://github.com/DOIO2022>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT(
+		KC_NUM, KC_PSLS, KC_PAST, KC_PMNS, RGB_MOD,
+		KC_P7,  KC_P8,   KC_P9,   KC_PPLS, RGB_TOG,
+		KC_P4,  KC_P5,   KC_P6,
+		KC_P1,  KC_P2,   KC_P3,   KC_PENT,
```

Comment:
white space issues: 
```suggestion
        KC_NUM, KC_PSLS, KC_PAST, KC_PMNS, RGB_MOD,
        KC_P7,  KC_P8,   KC_P9,   KC_PPLS, RGB_TOG,
        KC_P4,  KC_P5,   KC_P6,
        KC_P1,  KC_P2,   KC_P3,   KC_PENT,
```

Also, while it may not really matter here, would be better to align these with the community numpad layouts. Eg:

```c
        KC_NUM, KC_PSLS, KC_PAST, KC_PMNS, RGB_MOD,
        KC_P7,  KC_P8,   KC_P9,            RGB_TOG,
        KC_P4,  KC_P5,   KC_P6,   KC_PPLS,
        KC_P1,  KC_P2,   KC_P3,
        KC_P0,  KC_PDOT, KC_MUTE, KC_PENT
```